src/main/scala/dsptools/DspTester.scala

```diff
@@ -238,12 +242,22 @@ class DspTester[T <: Module](c: T,
     }
   }
 
+  def nearlyEqual(a: Double, b: Double): Boolean = {
+    val epsilon = 1e-12
+    val divisor: Double = if(b == 0.0) b.abs else epsilon
```

Comment:
I just realized this is wrong. First, I think you want to switch the true and false statements, and second of all I think it would be better to do:
`if (b.abs < epsilon) epsilon else b.asb`